update.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
+
+WORKING_DIR = File.dirname(__FILE__)
+
```

Comment:
Would love if this had an `-h` option to self document usage.

Reply:
I added some description of how to use and what to expect.  Let me know if you think we need more.